ftl_node_integration_tests.yaml

```diff
@@ -0,0 +1,38 @@
+steps:
+#Build node ftl image with bazel
+  - name:  'gcr.io/cloud-builders/bazel'
+    args: ['build', 'ftl:node_builder_image.tar']
+  - name:  'gcr.io/cloud-builders/docker'
+    args: ['load', '--input', 'bazel-bin/ftl/node_builder_image.tar']
+# 1. Test packages are installed correctly
+  # Build node image
+  - name:  'bazel/ftl:node_builder_image'
+    args: ['--base', 'gcr.io/google-appengine/nodejs:latest',
+           '--name', 'gcr.io/ftl-node-test/packages-test:latest',
+           '--directory', 'ftl/node/testdata/packages_test/']
+  # Pull docker image into local daemon 
+  - name:  'gcr.io/cloud-builders/docker'
+    args: ['pull', 'gcr.io/ftl-node-test/packages-test:latest']
+  # Run structure tests with bazel
+  - name:  'gcr.io/cloud-builders/bazel'
+    args: ['run', '--', 'structure_tests:go_default_test', '-driver', 'tar', '-image',
+          'gcr.io/ftl-node-test/packages-test:latest', '/workspace/ftl/node/testdata/packages_test/structure_test.yaml']
+  #Delete the image
+  - name:  'gcr.io/cloud-builders/gcloud'
+    args: ['container', 'images', 'delete', 'gcr.io/ftl-node-test/packages-test:latest']
+# 2. Test gcp-build script executes correctly
+  # Build node image
+  - name:  'bazel/ftl:node_builder_image'
+    args: ['--base', 'gcr.io/google-appengine/nodejs:latest',
+           '--name', 'gcr.io/ftl-node-test/gcp-build-test:latest',
+           '--directory', 'ftl/node/testdata/gcp_build_test/']
+  # Pull docker image into local daemon 
+  - name:  'gcr.io/cloud-builders/docker'
+    args: ['pull', 'gcr.io/ftl-node-test/gcp-build-test:latest']
+  # Run structure tests with bazel
+  - name:  'gcr.io/cloud-builders/bazel'
```

Comment:
Can you make a BUILD rule for these tests and run them with "bazel test" instead of this?